examples/nlp/machine_translation/create_tarred_parallel_dataset.py

```diff
@@ -111,6 +112,7 @@ def write_batches_to_tarfiles(
             data='/tmp/concat_dataset.txt',
             vocab_size=args.vocab_size,
             model=os.path.join(args.out_dir, 'tokenizer.%d.BPE.model' % (args.vocab_size)),
+            coverage=args.coverage,
```

Comment:
I am wondering if this should be supported by automatic tarred dataset creation @ericharper ?

Reply:
Happy to include it in `preproc_mt_data.py`

Reply:
That would be great!